awx/ui_next/src/components/PaginatedDataList/PaginatedDataList.jsx

```diff
@@ -82,10 +81,8 @@ class PaginatedDataList extends React.Component {
         ];
     const queryParams = parseQueryString(qsConfig, location.search);
 
-    const itemDisplayName = ucFirst(pluralize(itemName));
-    const itemDisplayNamePlural = ucFirst(
-      itemNamePlural || pluralize(itemName)
-    );
+    const itemDisplayName = ucFirst(itemName);
+    const itemDisplayNamePlural = ucFirst(itemName);
 
     const dataListLabel = i18n._(t`${itemDisplayName} List`);
```

Comment:
Note that these messages should NOT change depending on the number of items in the list. They should always be as coded here, e.g. "Job List" (singular), "Please add Jobs to populate this list" (plural), "No Jobs found" (plural).

HOWEVER -- we might be able to convert all to plural and still have it makes sense (Switch to "Jobs List"). Then we would only need the plural version of the word and could cut out a lot of this complexity. 🤔

Reply:
Yeah, there are also a couple of phrases that are passed to `Lookup` that should not be pluralized.

Reply:
So the way to code currently works is that if itemName is `!1` then itemName is plural.  So, if the `Job List` contains only 1 job then it would render as `Job List` otherwise there will be more than 1 job and it will be `Jobs List`, which makes sense to me.  However, if we always want it to be in the singular that can be done too.